### PR TITLE
Update Authorization dll version and fix tests

### PR DESCRIPTION
**Pull request by namratab**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@namratab__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (nbaliga). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by ljhljh235** [comment]
@azuresdkci add to whitelist

**Comment by ogail** [comment]
@azuresdkci retest this please

File: src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/RoleAssignmentTests.cs [hunk 1]
```diff
@@ -116,6 +117,9 @@ public void RaUserPermissions()
                                         .ResourceGroups
                                         .First();
 
+                    // Wait to allow newly created object changes to propagate
+                    Thread.Sleep(TimeSpan.FromSeconds(20));
```
Comment (stankovski) [inline src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/RoleAssignmentTests.cs]: Please use this code instead https://github.com/Azure/azure-powershell/blob/dev/src/Common/Commands.Common/TestMockSupport.cs

File: src/ResourceManager/StreamAnalytics/Commands.StreamAnalytics.Test/Commands.StreamAnalytics.Test.csproj [hunk 2]
```diff
@@ -55,14 +55,16 @@
     <Reference Include="Microsoft.Azure.Gallery">
       <HintPath>..\..\..\packages\Microsoft.Azure.Gallery.2.6.0-preview\lib\net40\Microsoft.Azure.Gallery.dll</HintPath>
     </Reference>
-    <Reference Include="Microsoft.Azure.Management.Authorization">
-      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Authorization.0.16.0-preview\lib\net40\Microsoft.Azure.Management.Authorization.dll</HintPath>
+    <Reference Include="Microsoft.Azure.Management.Authorization, Version=0.9.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```
Comment (ogail) [inline src/ResourceManager/StreamAnalytics/Commands.StreamAnalytics.Test/Commands.StreamAnalytics.Test.csproj]: Nit: edit the `csproj` files and remove all additions of `Version=0.9.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL"` to references

**Comment by ogail** [comment]
- verify that the authorization package version across all projects is the same
- update `Azure.Test.Framework` package (which is in hydra-specs-pr) to use the same authorization package used here

File: src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/RoleAssignmentTests.cs (hunk 1 above)
Comment (namratab) [inline src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/RoleAssignmentTests.cs]: Thanks fixed it.

File: src/ResourceManager/StreamAnalytics/Commands.StreamAnalytics.Test/Commands.StreamAnalytics.Test.csproj (hunk 2 above)
Comment (namratab) [inline src/ResourceManager/StreamAnalytics/Commands.StreamAnalytics.Test/Commands.StreamAnalytics.Test.csproj]: Done, removed.

**Comment by namratab** [comment]
1. Verified the Auhtorization version across all packages/projects is same
2. Could not find reference to Authorization in the Test.Framework project in hydra-specs-pr. Could you please point me to it ? And that would be a new/separate pull request correct ?

**Comment by ogail** [comment]
@namratab for #2 nevermind it depend on AAD lib not this one

**Pull request merged**